client/landing/jetpack-cloud/components/security-icon/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import okayIcon from './images/security-okay.svg';
+import errorIcon from './images/security-error.svg';
+
+interface Props {
+	icon: string;
+	className?: string;
+}
+
+function SecurityIcon( props: Props ) {
```

Comment:
Because it doesn't have a state, and by extending `Component` or `PureComponent` that adds a state and the lifecycle components. Also I feel it's easier to read. :)

Reply:
I also prefer the functional style and I encourage the use of hooks over classes/state :)

Reply:
Cool! I also prefer hooks!